src/sentry/integrations/pipeline.py

```diff
@@ -175,5 +177,12 @@ def _dialog_success(self, org_integration):
         return self._dialog_response(serialize(org_integration, self.request.user), True)
 
     def _dialog_response(self, data, success):
-        context = {"payload": {"success": success, "data": data}}
+        subdomain = self.fetch_state("subdomain")
```

Comment:
@dashed can we get the subdomain from the organization instead? I don't think there is a good way to set the pipeline state without doing it in every integration or doing some sort of refactor.